x-pack/plugins/uptime/public/state/index.ts

```diff
@@ -12,6 +12,13 @@ import { rootReducer } from './reducers';
 
 export type AppState = ReturnType<typeof rootReducer>;
 
+type ComposeType = typeof compose;
```

Comment:
This was copied from another plugin's solution for providing a global type for redux devtools.